app/src/modules/settings/routes/roles/collection.vue

```diff
@@ -154,7 +154,7 @@ async function fetchRoles() {
 			...response.map((role: any) => {
 				return {
 					...translate(role),
-					count: role.users[0]?.count.id || 0,
+					count: role.users.length || 0,
```

Comment:
Okay than something with the aggregation is not working correctly since the response from the request looks like this:

```json
{
    "data": [
        {
            "id": "b56a5bc1-534c-4419-a896-e58562bb3bb4",
            "name": "Administrator",
            "description": "$t:admin_description",
            "icon": "verified",
            "admin_access": true,
            "users": [
                "e62575ef-67e5-4061-aef4-bbf0e020a95a"
            ]
        }
    ]
}
```

Reply:
Hmmmm interesting! Could it be https://github.com/directus/directus/issues/17142 you're seeing?

Reply:
```javascript
const response = await fetchAll<any[]>(`/roles`, {
	params: {
		limit: -1,
		fields: ['id', 'name', 'description', 'icon', 'admin_access', 'users'],
		deep: {
			users: {
				_aggregate: { count: 'id' },
				_groupBy: ['role'],
				_sort: 'role',
				_limit: -1,
			},
		},
		sort: 'name',
	},
});
```

Cannot confirm this _groupBy and _sort is already used.